build/entrypoint.sh

```diff
@@ -2,34 +2,67 @@
 
 set -ex
 
-# Initialize
 if [[ $1 == init ]]; then
+    echo "The init command is deprecated. Go to our github repo for setup instructions."
+fi
+
+if [[ $HOME == "/" ]] then
+    echo "When running rootless, you must set a home dir as the HOME env var. We recommend /data. Make sure it is writable by the user running the container (currently id is $(id -u) and HOME is $HOME)."
+    exit 1
+fi
+
+# give friendly error if you don't have protonmail data
+if [[ `find $HOME | wc -l` == 1 ]]; then # 1 because find $HOME will always return $HOME
+    echo 'Protonmail does not seem to have been initialized yet. Enter the container with something like `docker exec -it <container_name>` and type "help" for instructions on how to set up the ProtonMail Bridge'
+    timeout 10s /protonmail/proton-bridge --noninteractive # this starts the bridge in non-interactive mode and kills it after 20 seconds, so we can populate the vault with default values and override them with the env variables in the later step.
```

Comment:
this exists with an error for me when running as non-root on a new volume:
```
ERRO[Aug 26 12:50:29.816] Could not load/create vault key               error="could not create keychain: no keychain"
ERRO[Aug 26 12:50:30.080] Failed to migrate old accounts                error="failed to create keychain: no keychain"
WARN[Aug 26 12:50:30.085] The vault key could not be retrieved; the vault will not be encrypted 
```

As you have `set -e`, `entrypoint.sh` exits, i.e. the container fails.

Replacing this with `timeout 10s /protonmail/proton-bridge --noninteractive || /bin/true` is a hacky way to avoid entrypoint.sh to stop there.